config/rbac/overlays/openshift/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+bases:
+- ../../base
```

Comment:
This could just be `../..` to keep the same directory layout.

Reply:
see other comment